sql/moz-fx-data-shared-prod/relay_derived/active_subscriptions_live/view.sql

```diff
@@ -0,0 +1,49 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.relay_derived.active_subscriptions_live`
+AS
+WITH subscriptions AS (
+  SELECT
+    *,
+    TO_JSON_STRING(promotion_codes) AS json_promotion_codes
+  FROM
+    mozdata.relay.subscriptions
+)
+SELECT
+  active_subscription_ids.active_date,
+  subscriptions.plan_id,
+  subscriptions.status,
```

Comment:
> A couple of additional changes to make:
> 
>     * This is missing a `schema.yaml` file for `relay_derived.active_subscription_ids_v1`.
> 
>     * `sql/moz-fx-data-shared-prod/relay_derived/subscriptions_v1/query.sql` should be added to the ["Access Denied" section of `bigquery_etl/dryrun.py`](https://github.com/mozilla/bigquery-etl/blob/e11d5fc637fc2b988e25fa9d9907995253bf1164/bigquery_etl/dryrun.py#L35), because the account that does the dry runs won't have permission to query the Stripe tables.

 `schema.yaml` file for `relay_derived.active_subscription_ids_v1` is created but I noticed that `mozilla_vpn_derived.active_subscription_ids_v1` is missing `schema.yaml`. When  do you create `schema.yaml` and when you don't? 
 
 Also `sql/moz-fx-data-shared-prod/relay_derived/subscriptions_v1/query.sql`  was added to the access denied list.

Reply:
> When do you create `schema.yaml` and when you don't?

These days with how the deployment of new tables is automated every table should have a `schema.yaml` file.  Without a `schema.yaml` file it will try to dry-run the query to determine the schema, but if the dry-run fails (e.g. due to permissions) then the table deployment will fail.